RedHat-UX/red-hat-design-system
# feat(accordion): remove BaseAccordion import from PFE

## What I did

1.  Moved all the code in for `BaseAccordion` (that I could), `BaseAccordionHeader`, and `BaseAccordionPanel`.  

I was able to remove all the need for the code for Header and Panels base classes, however the Header needs to be able to see if what called it was an Accordion component (and if so, send an event) and if I moved that code out of a BaseClass we get a circular dependency.  Went back and forth on it but currently that's the only function still remaining in the BaseAccordion and it's so the header can check if it's a `BaseAccordion` class instead of an `RhAccordion` class.  


## Testing Instructions

1.  Open up the demos for `RhAccordion` and verify that there are no regressions in the code.  Everything should work normally. 

## Notes to Reviewers
